### PR TITLE
Lt 936 gov users

### PR DESCRIPTION
**Pull request by lukesa**

### CODE REVIEW

File: gov_users/tests/tests_view.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    urlpatterns = [
+        path('gov-users/', include('gov_users.urls')),
+        path('organisations/', include('organisations.urls'))
+    ]
+
+    def setUp(self):
+        super().setUp()
+
+    def tests_get(self):
+        GovUser(email='test2@mail.com',first_name='John',last_name='Smith',team=self.team).save()
```
[the remark this answers is not in the document]
Comment (cstobart) [reply gov_users/tests/tests_view.py]: I'd rather keep additional objects confined to single test classes

File: teams/tests/tests_list_user_by_team.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+class UserByTeamListTests(DataTestClient):
+
+    def setUp(self):
+        super().setUp()
+
+    def tests_user_by_team_list(self):
+        team = Team.objects.get(name='Admin')
+        team2 = Team(name='Second')
+        team2.save()
+        govuser1 = GovUser(email='test2@mail.com', first_name='John', last_name='Smith', team=self.team)
```
[the remark this answers is not in the document]
Comment (cstobart) [reply teams/tests/tests_list_user_by_team.py]: Would rather keep additional objects confined to single test classes

File: gov_users/views.py [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+
+class GovUserList(APIView):
+    authentication_classes = (GovAuthentication,)
+
+    def get(self, request):
+        serializer = GovUserSerializer(GovUser.objects.all(), many=True)
+        return JsonResponse(data={'gov_users': serializer.data}, safe=False)
+
+    def post(self, request):
```
[the remark this answers is not in the document]
Comment (lukesa) [reply gov_users/views.py]: still needs swagger decorator to reference GovUserSerializer

File: teams/views.py [hunk 4]
```diff
@@ -79,3 +84,13 @@ def put(self, request, pk):
             return JsonResponse(data={'team': serializer.data})
         return JsonResponse(data={'errors': serializer.errors},
                             status=400)
+
+
+class UsersByTeamsList(APIView):
+    authentication_classes = (GovAuthentication,)
+
+    def get(self, request, pk):
```
[the remark this answers is not in the document]
Comment (lukesa) [reply teams/views.py]: get method still needs docstring

File: docker.env [hunk 5]
```diff
@@ -2,3 +2,6 @@ DEBUG=True
 DATABASE_URL=postgres://postgres:password@db:5432/postgres
 PORT=8100
 ALLOWED_HOSTS='["*"]'
+
+# Seed
+SEED_USERS='["christopher.stobart@digitial.trade.gov.uk","example@example.com"]'
```
Comment (jfaracik) [inline docker.env]: YOUR EMAIL IS HERE

File: gov_users/libraries/token_to_user_pk.py [hunk 6]
```diff
@@ -0,0 +1,5 @@
+from gov_users.models import GovUser
+
+
+def token_to_user_pk(token):
```
Comment (jfaracik) [inline gov_users/libraries/token_to_user_pk.py]: what on earth

Comment (jfaracik) [reply gov_users/libraries/token_to_user_pk.py]: what on earth x 2

**Pull request merged**